app/Http/Controllers/AssetModelsController.php

```diff
@@ -227,10 +237,7 @@ public function postEdit($modelId = null)
         $model->modelno             = e(Input::get('modelno'));
         $model->manufacturer_id     = e(Input::get('manufacturer_id'));
         $model->category_id         = e(Input::get('category_id'));
-        $model->note                = e(Input::get('note'));
-
-        $model->requestable = Input::has('requestable');
```

Comment:
Did you remove the requestable option, or is it just moved somewhere else in this method?